### DISCUSSION TITLE
Can't get the Windows 7 like Aero on Windows 10 22h2

### DISCUSSION THREAD

**Discussion by mjmc12342025** [ALTaleX531/OpenGlass#35] [Q&A]
I just cant get the Windows 7 aero effect / look on Windows 10 22h2 with OpenGlass Legacy. Can someone explain me how to get it or provide some config? I really want this effect from the preview, but for me its only normal blur ;(
![349774758-9d52fb58-e6b8-438e-8801-28afc19ecdbc](https://github.com/user-attachments/assets/7283ffb8-054b-4246-ad82-07daa58b9186)
![349774713-87bd340e-f60f-427b-83dd-d5470af3628a](https://github.com/user-attachments/assets/6fda6903-cf23-4c69-b2de-5de36c6f6913)

**Comment by DAPOPOBEFASTONYOAZZ** [comment]
I know it's a bit late, but I can provide my config that shows all the registry keys you need to change to get both Windows Vista and Windows 7 exact colors. I also have a theme that makes it look identical to 7 or Vista, whichever window borders you choose to use (white vs cyan, respectively.)

You'll need WinPaletter to make the initial color adjustments, as any color adjustments you make in the registry are immediately reset and will not take effect. Version 1.0.9.3 is compatible with the attached color themes offered in Windows Vista and Windows 7, using Windows 8's default purple as a metro/fluent UI fallback color.

The registry colors text file will display the exact registry keys and the values you need to tweak after applying the desired color theme for 100% accurate effect.

As a note, you may use custom colors, but if you're trying to tweak the opacity to a desired amount, I can only provide Vista's calculation at this time since Windows 7's method includes multiplication blending, but the default color values were extracted from the registry. If you want a color with no multiplication effect, you'll use GlassType: 0, which is what it would look like in Windows Vista. You can change the GlassOpacity DWORD to determine color intensity. Windows Vista and 7's max intensity by the slider was 85 (hex value 55.) If you are using GlassType: 0, then you will not need to make any additional adjustments to calculate intensity to any other registry keys aside from AccentColorInactive. Any color you choose must have 80 at the beginning of that registry value so it will go to half intensity when the window is inactive.

[WinPaletter Themes.zip](https://github.com/user-attachments/files/17832119/WinPaletter.Themes.zip)
[registry colors.txt](https://github.com/user-attachments/files/17832164/registry.colors.txt)

**Reply by Milicent-Heeler** [reply]
Please provide you config. Thanks.

**Reply by DAPOPOBEFASTONYOAZZ** [reply]
Please see my comment at #63 for a more detailed description and updated files for how to set up your config. I can only provide what I have already, I cannot provide registry configs for you. You will have to enter the registry keys on your own. I don't want to risk borking your system for a registry configuration that works for me, but not for you.

**Reply by Milicent-Heeler** [reply]
> Please see my comment at #63 for a more detailed description and updated files for how to set up your config. I can only provide what I have already, I cannot provide registry configs for you. You will have to enter the registry keys on your own. I don't want to risk borking your system for a registry configuration that works for me, but not for you.

What branch do you have? Dcomp or legacy?
